### PR TITLE
Update software versions for RTD

### PR DESCRIPTION
**Pull request by nabobalis**
I might grow this PR before its merged.

### CODE REVIEW

File: {{ cookiecutter.package_name }}/pyproject.toml [hunk 1]
```diff
@@ -54,10 +54,16 @@ zip-safe = false
 include-package-data = true
 
 [tool.setuptools.packages.find]
+include = ["{{ cookiecutter.module_name }}*"]
 {%- if cookiecutter.enable_dynamic_dev_versions == 'y' %}
 exclude = ["{{ cookiecutter.module_name }}._dev*"]
 {%- endif %}
 
+{% if cookiecutter.use_compiled_extensions == 'y' %}
```
Comment (nabobalis) [inline {{ cookiecutter.package_name }}/pyproject.toml:62]:
```suggestion
{%- if cookiecutter.use_compiled_extensions == 'y' %}
```

File: {{ cookiecutter.package_name }}/pyproject.toml [hunk 2]
```diff
@@ -54,10 +54,14 @@ zip-safe = false
 include-package-data = true
 
 [tool.setuptools.packages.find]
+include = ["{{ cookiecutter.module_name }}*"]
```
Comment (nabobalis) [inline {{ cookiecutter.package_name }}/pyproject.toml:57]: I checked this entire block rendered correctly for the three test cases.

It has the Nabil approved amount of line breaks.

**Comment by Cadair** [comment]
Looks good at the moment, let me know when you wanna merge it. If we merge it this week it will be in next Monday's updates.

**Comment by Cadair** [comment]
I suspect the pyproject.toml bit needs to be in the OA template tbh, but I can do that once we merge.

**Comment by nabobalis** [comment]
This is as far as I will go.

**Review by Cadair — APPROVED** [review]

**Pull request merged**